### PR TITLE
feat: support updating a release on a release or hotfix branch (CORE-1465)

### PR DESCRIPTION
**Pull request by robaone-redshelf**
### Description:

The action is not finding any tickets

### Ticket:

https://virdocs.atlassian.net/browse/CORE-1465

### Changes: (complexity: medium)

- [ ] Add support for updating a release after it is created
- [ ] Add support for tagging tickets that are in a release or hotfix branch

### Validation:

- [ ] See https://github.com/VirdocsSoftware/rsm-monorepo/actions/runs/11804594984/job/32885522682

### CODE REVIEW

File: .github/actions/tag-jira-release/action.yml [hunk 1]
```diff
@@ -22,6 +22,10 @@ inputs:
     description: "The description of the release"
     required: false
     default: "Next awesome release"
+  is-release:
+    description: "Whether to create a release"
```
Comment (robaone-redshelf) [inline .github/actions/tag-jira-release/action.yml:26]: Is this a release branch?

File: .github/actions/tag-jira-release/jira_tag_tickets.sh [hunk 2]
```diff
@@ -110,7 +110,13 @@ function existing_version() {
 }
 
 NEXT_PREDICTED_VERSION="$($GIT_PREDICT_NEXT_VERSION_PATH)"
-TARGET_BRANCH="$(get_target_branch $NEXT_PREDICTED_VERSION)"
+# if this branch does not start with `release/` or `hotfix`
+# then find the target branch
```
[the remark this answers is not in the document]
Comment (robaone-redshelf) [reply .github/actions/tag-jira-release/jira_tag_tickets.sh]: Addressed in [c8a9447](https://github.com/VirdocsSoftware/github-actions/pull/13/commits/c8a9447ffce7cd6eb84401967b568aa5b47fdb7e)

File: .github/actions/tag-jira-release/entrypoint.sh [hunk 3]
```diff
@@ -5,5 +5,5 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 if [ "$DRY_RUN" == "true" ]; then
   $SCRIPT_DIR/jira_tag_tickets.sh --dry-run
 else
-  $SCRIPT_DIR/jira_tag_tickets.sh
+  bash -xv $SCRIPT_DIR/jira_tag_tickets.sh
```
[the remark this answers is not in the document]
Comment (robaone-redshelf) [reply .github/actions/tag-jira-release/entrypoint.sh]: Addressed in [c8a9447](https://github.com/VirdocsSoftware/github-actions/pull/13/commits/c8a9447ffce7cd6eb84401967b568aa5b47fdb7e)

File: .github/actions/tag-jira-release/action.yml (hunk 1 above)
Comment (robaone-redshelf) [reply .github/actions/tag-jira-release/action.yml]: Addressed in [c8a9447](https://github.com/VirdocsSoftware/github-actions/pull/13/commits/c8a9447ffce7cd6eb84401967b568aa5b47fdb7e)

**Review by ottobarnes — APPROVED** [review]

**Pull request merged**